Source/WebCore/html/RangeInputType.cpp

```diff
@@ -172,17 +172,13 @@ void RangeInputType::handleTouchEvent(TouchEvent& event)
         typedSliderThumbElement().setPositionFromPoint(touches->item(0)->absoluteLocation());
         event.setDefaultHandled();
     }
-#else
-    UNUSED_PARAM(event);
 #endif
 }
 
-#if ENABLE(TOUCH_SLIDER)
 bool RangeInputType::hasTouchEventHandler() const
 {
     return true;
 }
-#endif
```

Comment:
I think here instead of removing the conditional it should be replaced with one for: `#if !PLATFORM(IOS_FAMILY)` to match how this function is defined on the header file at `RangeInputType.h`